esy-lib/Fs.re

```diff
@@ -209,6 +209,7 @@ let fold =
       let%lwt stat = Lwt_unix.lstat(spath);
       switch (stat.Unix.st_kind) {
       | Unix.S_DIR =>
+        let%lwt _ = f(acc, path, stat);
```

Comment:
I did not understand this.

Reply:
the `Fs.fold` applied reducer only on files (leaf nodes) it ignore directories
this line applies the reducer on directories as well

Reply:
Thanks